### PR TITLE
ODC-2518

### PR DESCRIPTION
**Pull request by cervandt**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,125 @@
+buildscript {
+    project.ext['artifactoryDomainName'] = 'https://artifactory.viralgains.com/artifactory'
```
Comment (mlubavin-vg) [inline build.gradle]: by putting our artifactory credentials in here, we'll never be able do a PR back to the project

Comment (cervandt) [reply build.gradle]: If I remove it , will it work?

Comment (mlubavin-vg) [reply build.gradle]: actually its just for pulling dependencies from artifactory, so we don't need it. please remove

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+group 'com.mediamath'
+version = '1.2.1'
+sourceCompatibility = 1.8
+
+//don't repackage the jar as an executable, this is just a simple library jar
+jar.enabled = true
+
+repositories {
+    mavenCentral()
+    maven {
```
Comment (mlubavin-vg) [inline build.gradle]: also this maven block

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        credentials {
+            username "$artifactoryUsername"
+            password "$artifactoryPassword"
+        }
+        url "$artifactoryDomainName/viralgains-libs"
+    }
+}
+
+//for dynamically changing modules, such as SNAPSHOT builds, have gradle recheck each time (needed for our odyssey-common)
+configurations.all {
```
Comment (mlubavin-vg) [inline build.gradle]: and the configurations.all

**Pull request merged**